shared/api/metrics.go

```diff
@@ -0,0 +1,95 @@
+package api
```

Comment:
We need the swagger examples added for all of the fields in this file

Reply:
Hmm, actually, maybe we don't. The actual metrics API is just plain text, so all those structs are only for internal use between LXD and the agent, right?

If so, then this shouldn't be in `shared/api` in the first place.

Reply:
It's for internal use, yes, but there's still a `GetMetrics()` function is the `client` package. Should I still move it to `lxd/metrics`?

Reply:
I've moved it to `lxd/metrics`.